meson.build

```
@@ -29,6 +29,10 @@ add_project_arguments(
   ],
   language: 'cpp')
 
+if cpp_compiler.check_header('execinfo.h')
+  add_project_arguments('-DHAS_LIBEXECINFO', language: 'cpp')
```

Obsolete macro name after 81115d5f0e7e.
```suggestion
  add_project_arguments('-DHAS_EXECINFO', language: 'cpp')
```